embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/comms/delivery/EmbraceCacheService.kt

```diff
@@ -194,30 +194,54 @@ internal class EmbraceCacheService(
     }
 
     override fun writeSession(name: String, sessionMessage: SessionMessage) {
+        safeFileWrite(name) { tempFile ->
+            serializer.toJson(sessionMessage, SessionMessage::class.java, tempFile.outputStream())
+        }
+    }
+
+    override fun transformSession(name: String, transformer: (SessionMessage) -> SessionMessage) {
```

Comment:
This did not change. The diff looks weird because I put the private helper function below the two overridden ones, and GitHub hub thinks is the best way to diff for some reason.